### PR TITLE
fix(pickers): improve responsiveness when using longer strings

### PR DESCRIPTION
**Pull request by jeremypress**
Before: 
Screen Shot 2019-04-23 at 3 12 23 PM

After: 
Screen Shot 2019-04-24 at 3 34 55 PM

Empty state text
Before:
Screen Shot 2019-04-24 at 9 59 31 AM

After:
Screen Shot 2019-04-24 at 9 58 04 AM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

File: src/elements/common/empty-state/EmptyState.scss [hunk 1]
```diff
@@ -4,4 +4,5 @@
     flex: 1;
     flex-direction: column;
     justify-content: center;
+    text-align: center;
```
Comment (jstoffan) [inline src/elements/common/empty-state/EmptyState.scss]: Unrelated?

File: src/elements/content-picker/Footer.scss [hunk 2]
```diff
@@ -9,11 +9,23 @@
...
+        min-width: 0;
+    }
+
     .bcp-selected {
+        flex: 0 0 75px;
         margin: 0;
+        width: 100%;
 
         .btn-content {
-            display: flex;
```
Comment (jstoffan) [inline src/elements/content-picker/Footer.scss]: Is everything inside the selected button, including the red file limit text (`(2 max)`), still vertically centered with this change?

File: src/elements/content-picker/Footer.scss [hunk 3]
```diff
@@ -9,11 +9,23 @@
     justify-content: space-between;
     padding: 0 20px;
 
+    .bcp-footer-left {
+        display: flex;
+        flex: 1;
+        min-width: 0;
```
Comment (jstoffan) [inline src/elements/content-picker/Footer.scss]: Is this necessary? Is it overriding something?

File: src/elements/content-picker/Footer.scss [hunk 4]
```diff
@@ -9,11 +9,23 @@
...
+    }
+
     .bcp-selected {
+        flex: 0 0 75px;
         margin: 0;
+        width: 100%;
```
Comment (jstoffan) [inline src/elements/content-picker/Footer.scss]: Generally not a great idea to mix flex and width. Should the basis or the width take precedence here?

File: src/elements/content-picker/ItemList.js [hunk 5]
```diff
@@ -120,7 +120,7 @@ const ItemList = ({
                             height={height}
                             disableHeader
                             headerHeight={0}
-                            rowHeight={50}
+                            rowHeight={65}
```
Comment (jstoffan) [inline src/elements/content-picker/ItemList.js]: Does this still look okay in wider resolutions and/or with single-line items? Can you post a screenshot?

File: src/elements/common/empty-state/EmptyState.scss (hunk 1 above)
Comment (jeremypress) [reply src/elements/common/empty-state/EmptyState.scss]: unrelated but fixes text alignment for the empty state at small widths. I can make this a separate change if you'd like.

File: src/elements/content-picker/ItemList.js (hunk 5 above)
Comment (jeremypress) [reply src/elements/content-picker/ItemList.js]: Added screenshots to the PR description.

File: src/elements/content-picker/Footer.scss (hunk 3 above)
Comment (jeremypress) [reply src/elements/content-picker/Footer.scss]: The button won't resize without it: following advice from stack overflow / https://css-tricks.com/flexbox-truncated-text/

**Comment by ConradJChan** [comment]
The 65px looks pretty spacious on a wider screen, should we instead use the `isSmall` prop and add some logic around reducing the content showed on each line to avoid increasing the row height?

File: src/elements/content-picker/Footer.scss (hunk 2 above)
Comment (jeremypress) [reply src/elements/content-picker/Footer.scss]: looks like the centering is off, I'll look for a solution.

**Pull request merged**